airbyte-integrations/connectors/source-hubspot/source_hubspot/schemas/workflows.json

```diff
@@ -158,6 +158,7 @@
       "properties": {
         "sourceApplication": {
           "description": "Application details of the workflow update source",
+          "type": ["null", "object"],
```

Comment:
Out of curiosity, what is the default type if this is not provided? Should this have been caught by CATs?

Reply:
This is a great question.

The data structure remains the same, but the schema was missing these types only. 
I wonder why the CAT didn't catch such things, 
I assume we don't have the `type` field presence check yet for the schemas, as a separate testing scenario for the CAT, thus we can easily bypass such things. 

Correct me if I'm wrong here.

Reply:
I would definitely expect [this test](https://github.com/airbytehq/airbyte/blob/f1d12954b8ac995dc2ac6205617af98a24bc351f/airbyte-ci/connectors/live-tests/src/live_tests/validation_tests/test_discover.py#L50) to fail in that case. Maybe we haven't configured the validator properly..?

Reply:
`type` field, is optional for the JsonSchema, in general

Reply:
What is the default value? I could not find this information. If it is the case, we either need to have destinations support this or update our CATs. The latter seems to build more leverage and seems easier to implement too

Reply:
I think the default value, in this case, depends on the `destination` used, there is no `default` value for the source, in this case, it's a matter of how the destination treats the `missing` types, for instance, `BigQuery` raises and Exception that there is no type, but expected `any of Null, Object` based on the actual data type faced.